x-pack/plugins/maps/public/classes/layers/vector_tile_layer/vector_tile_layer.js

```diff
@@ -56,7 +56,7 @@ export class VectorTileLayer extends TileLayer {
 
     const requestToken = Symbol(`layer-source-refresh:${this.getId()} - source`);
     try {
-      startLoading(SOURCE_DATA_REQUEST_ID, requestToken, dataFilters);
+      startLoading(SOURCE_DATA_REQUEST_ID, requestToken, { dataFilters, ...nextMeta });
```

Comment:
Can you remove nextMeta from call to stopLoading. Its not needed in stopLoading because its already set. meta in stopLoading is there to add additional meta about the response. In this case, there is no new meta from the response.